HW#7/Cows and bull.py

```diff
@@ -0,0 +1,23 @@
+# Быки и коровы
+n = input('Загадайте 4-значное число: ')
+b = 0
```

Comment:
It is better to use an explicit name: 
- `bulls` instead of `b`
- `cows` instead of `c`